src/main/groovy/com/mindera/gradle/versioncode/VersionCodePlugin.groovy

```diff
@@ -15,11 +15,13 @@ class VersionCodePlugin implements Plugin<Project> {
 
         project.task('incrementVersionCode') << {
 
+            def appId = project.hasProperty('appId') ? project.appId : project.appVersionCode.appId
```

Comment:
Can you explain the difference between this too?
And to the user does it change anything? This appId parameter is optional and override what we define in the property? I think we should be more explicit about that in the readme.